bin/node/runtime/src/lib.rs

```diff
@@ -2148,15 +2148,18 @@ impl_runtime_apis! {
 		) -> Weight {
 			log::info!(
 				target: "node-runtime",
-				"try-runtime: executing block {:?} / root checks: {:?} / try-state-select: {:?}",
+				"try-runtime: executing block {:?} / state root check: {:?} / signature check: {:?} / try-state-select: {:?}",
 				block.header.hash(),
 				state_root_check,
+				false,
 				select,
 			);
```

Comment:
Why isn't this printed in `try_execute_block`?

Reply:
Yeah seems a bit too noisy, will remove.